Symfony/CS/Tests/Fixer/PSR2/UnneededControlParenthesesFixerTest.php

```diff
@@ -0,0 +1,363 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\PSR2;
+
+use Symfony\CS\Fixer\PSR2\UnneededControlParenthesesFixer;
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ *
+ * @internal
+ */
+final class UnneededControlParenthesesFixerTest extends AbstractFixerTestBase
+{
+    public function testConfigure()
+    {
+        $loopsProperty = new \ReflectionProperty('\Symfony\CS\Fixer\PSR2\UnneededControlParenthesesFixer', 'loops');
```

Comment:
No slash should be used on `Symfony\CS\Fixer\PSR2\UnneededControlParenthesesFixer`. PHP just trims it off under the hood.